app/routes/wallet/credentials.py

```diff
@@ -45,12 +75,31 @@ async def list_credentials(
     return results
 
 
-@router.get("/{credential_id}", response_model=IndyCredInfo)
+@router.get(
+    "/{credential_id}",
+    response_model=IndyCredInfo,
+    summary="Fetch a credential by ID",
+)
 async def get_credential_record(
     credential_id: str,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> IndyCredInfo:
-    """Fetch a specific credential by ID."""
+    """
+    Fetch a specific credential by credential ID
+    ---
+
+    The referent and credential_id are the duplicates of each other.
```

Comment:
Not a necessary detail. What's more important is the `referent` is deprecated. But remember that these descriptions must be relevant to the function of the route.